test/entry-io.spec.js

```diff
@@ -13,44 +13,43 @@ const dataDir = './ipfs/tests/fetch'
 
 const repoConf = {
   storageBackends: {
-    blocks: DatastoreLevel,
-  },
+    blocks: DatastoreLevel
+  }
 }
 
-let ipfs, ipfsDaemon
+let ipfs
 
 const last = arr => arr[arr.length - 1]
 
 apis.forEach((IPFS) => {
-  describe('Entry - Persistency', function() {
+  describe('Entry - Persistency', function () {
     this.timeout(20000)
 
     before((done) => {
       rmrf.sync(dataDir)
-      ipfs = new IPFS({ 
+      ipfs = new IPFS({
         repo: new IPFSRepo(dataDir, repoConf),
         start: true,
         EXPERIMENTAL: {
           pubsub: true,
           dht: false,
-          sharding: false,
-        },
+          sharding: false
+        }
       })
       ipfs.on('error', done)
       ipfs.on('ready', () => done())
     })
 
     after(async () => {
-      if (ipfs) 
-        await ipfs.stop()
+      if (ipfs) { await ipfs.stop() }
     })
 
     it('log with one entry', async () => {
       let log = new Log(ipfs, 'X')
       await log.append('one')
       const hash = log.values[0].hash
       const res = await EntryIO.fetchAll(ipfs, hash, 1)
-      assert.equal(res.length, 1)
+      assert.strictEqual(res.length, 1)
```

Comment:
That's because of the rules enforced by `eslint-plugin-node` that is part of standard js config. The rule in question is this one [node/no-deprecated-apis](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-deprecated-api.md)

Reply:
`assert.equal` is going to be deprecated in favour of `assert.strictEqual`

Reply:
Got it. Didn't know that, thanks for the explanation! ❤️